keras_nlp/layers/modeling/masked_lm_head.py

```diff
@@ -157,12 +156,13 @@ def build(self, inputs_shape, masked_positions_shape=None):
         self._layer_norm = keras.layers.LayerNormalization(
             epsilon=self.layer_norm_epsilon,
         )
-        if masked_positions_shape:
-            gather_length = masked_positions_shape[1]
-            shape = (inputs_shape[0], gather_length, inputs_shape[-1])
-            self._dense.build(shape)
-            shape = (inputs_shape[0], gather_length, feature_size)
-            self._layer_norm.build(shape)
+        # The gather length does not affect any of our built variables, so
```

Comment:
Does using `None` break this? seems a bit less awkward but if it doesn't work then so be it

Reply:
Seems to work!